test/connection-pool.test.js

```diff
@@ -1,16 +1,13 @@
 var assert = require('assert');
 var Request = require('tedious').Request;
+var fs = require('fs');
 var ConnectionPool = require('../lib/connection-pool');
 
-var connectionConfig = {
-    userName: 'test',
-    password: 'test',
-    server: 'dev1',
-    options: {
-        appName: 'pool-test',
-        database: 'test'
-    }
-};
+function getConfig() {
+    var config = JSON.parse(fs.readFileSync(process.env.HOME + '/.tedious/test-connection-pool.json', 'utf8')).config;
```

Comment:
part of the reason I didn't use require and chose to reread the config each time is that the connectionConfig object gets populated by default settings.  By rereading the config each time it doesn't rely on any settings that were added the the connectionConfig object from previous tests.